src/libs/antares/CMakeLists.txt

```diff
@@ -567,6 +564,7 @@ target_link_libraries(libantares-core
 		libantares-core-calendar
 		libantares-solver-variable-info
         result_writer
+        ortools::ortools

```

Comment:
This shouldn't be necessary if we keep a `std::string` in class `Parameters`.

It was necessary when we kept an `MPSolver::OptimizationType`